crates/sui-types/src/messages.rs

```diff
@@ -2647,6 +2649,16 @@ impl Display for ExecutionFailureStatus {
                 f,
                 "Publish Error, Duplicate Module. More than one module with a given name."
             ),
+            ExecutionFailureStatus::MissingExplicitDependencies => write!(
```

Comment:
@tnowacki went through here like a bull in a china shop, so brace yourself for more merge conflicts.